packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -404,6 +405,15 @@ export interface GridOptions {
    * You can find all the translation keys supported in [the source](https://github.com/mui-org/material-ui-x/blob/HEAD/packages/grid/_modules_/grid/constants/localeTextConstants.ts) in the GitHub repository.
    */
   localeText: Partial<GridLocaleText>;
+  /**
+   * Callback fired when scrolling to the bottom of the grid viewport.
+   * @param param
+   */
+  onRowsScrollEnd?: (params: GridRowScrollEndParams) => void;
+  /**
+   * Set the area at the bottom of the grid viewport where onRowsScrollEnd is called.
+   */
+  scrollEndThresholdHeight: number;
```

Comment:
Does it need to include "height" in the name?

```suggestion
  scrollEndThreshold: number;
```